README.rst

```diff
@@ -36,15 +36,17 @@ Example:
         username: deedee
         email: deedee@example.com
 
-    Profile:
+    - Profile:
       - user: dee
         name: Douglas
 
-    Group:
+    - Group:
       - name: Ramones
         members: [joey.profile, dee.profile]
 
--  The root of YAML contain ``mapper`` names i.e. ``User``
+-  The root of YAML contain ``mapper`` names e.g. ``User``, ``Profile`` etc
+-  The ``mapper`` names should be a ``list`` to enforce ordering e.g. ``- User``, ``- Profile``
+-  Order of ``mapper`` names should follow relationship (``__key__``) dependencies
```

Comment:
at this point `__key__` was not introduced yet. IMO no need to mention this about ordering, lists are ordered...